context/design-tokens/README.md

```diff
@@ -2,156 +2,205 @@
+  - **public:** means that the design token will be generated by SD
+  - **deprecated:** means if it’s deprecated there would be a message pointing to new value
+  - **documented:** means that the design token will show up in the tokens documentation
+  - **rxperimental:** if the token is part of a trial, user research, and subject to change.
+  - **SassVariable:** A Sass variable is generated by SD and the value is shown in the documentation
+  - **CSSCustomProperty:** A CSS Custom Property is generated by SD and the value is shown in the documentation.
+
+#### Example
+
+```json
 {
-	"breakpoints": {
-		"xs": {
-			"value": "320px",
-			"comment": "mobile"
-		},
-		"md": {
-			"value": "768px",
-			"comment": "tablet"
-		},
-		"lg": {
-			"value": "1024px",
-			"comment": "small desktop"
-		}
-	}
+    "aspect-ratio": {
+        "1x1": {
+            "name": "1x1",
+            "value": "1",
+            "description": "This is to make an item square",
+            "type": "ratio",
+            "category": "aspect-ratio",
+            "themeable": false,
+            "meta": {
+                "public": true,
+                "deprecated": false,
+                "documented": true,
+                "experimental": false,
+                "SassVariable": true,
+                "CSSCustomProperty": false
+            }
+        }
+    }
 }
 ```
+## Design Token Tiering System
+
+There are three tiers of Design Tokens:
+
+- `literal`: The base set of tokens that are used to build the rest of the tokens.
+- `alias`: The set of tokens that are used to build the rest of the tokens.
+- `component`: The set of tokens that are used to build the rest of the tokens.
+
+### Literal Design Tokens
+These are the ‘locked’ tokens, that hold a value that should not change. The literal Design Tokens are defined in the `tokens/literal` folder. The literal Design Tokens are used to create the Alias Design Tokens.
 
-#todo
-- These are also used to generate the styling utility classes
-- These will be expanded upon with values for `z-index`, `box-shadow`, `border` properties (and more).
-- - -
 ### Alias Design Tokens
-Alias Design Tokens are ‘locked’ tokens that hold a literal design tokens, and not a value.
-#todo
-- This is part of 'phase 2' of the Design Tokens Epic
+These are the ‘unlocked’ tokens, that hold a value that can change. The Alias Design Tokens are defined in the `tokens/alias` folder. The Alias Design Tokens are used to create the Component Design Tokens.
 
 ### Component Design Tokens
-Component Design Tokens are 'locked' tokens that hold a Alias design token, and not a literal token or value.
-#todo
-- This is part of 'phase 3' of the Design Tokens Epic
+These are the ‘unlocked’ tokens, that hold a value that can change. The Component Design Tokens are defined in the `tokens/components` folder. The Component Design Tokens are used to create the Component Design Tokens.
```

Comment:
The second sentence doesn't make sense to me... is it a mistake?

Reply:
I've updated this so it makes sense (hopefully)